6.7.2:release-notes (#3841)
sensu/sensu-docs: content/sensu-go/6.7/release-notes.md

---

## 6.7.2 release notes

**May 11, 2022** &mdash; The latest release of Sensu Go, version 6.7.2, is now available for download.

> Sensu Go 6.7.2 delivers several improvements and fixes to the sensu-backend stability and the postgres datastore.

Reviewer: ```suggestion
Sensu Go 6.7.2 includes a fix for sensu-backend stability and adds an active poller for PostgreSQL config changes.
We've also improved the Sensu Plus modal in the web UI.
```

Read the [upgrade guide][1] to upgrade Sensu to version 6.7.2.

**IMPROVEMENTS**

> - Added the etcd-unsafe-no-fsync backend configuration option, making it possible to run a sensu-backend with an embedded etcd node for testing and development without placing lots of load on the file system.

Reviewer: ```suggestion
- Added the etcd-unsafe-no-fsync backend configuration option, which makes it possible to run sensu-backend with an embedded etcd node for testing and development without placing too much load on the file system.
```

> - Added the etcd-unsafe-no-fsync backend configuration option, which makes it possible to run sensu-backend with an embedded etcd node for testing and development without placing too much load on the file system.

Author: ```suggestion
- ([Commercial feature][268]) In the web UI, we updated the Sensu Plus dialog copy to warn users not to use the form when they have an existing Sumo Logic account.
- ([Commercial feature][268]) In the web UI, Sensu Plus post setup dialog Source URL copy button now uses a toast notification.
- Added the etcd-unsafe-no-fsync backend configuration option, which makes it possible to run sensu-backend with an embedded etcd node for testing and development without placing too much load on the file system.
```

- Upgraded etcd version from 3.5.2 to 3.5.4.

**FIXES**

> - ([Commercial feature][268]) Fixed a backend crash when pruning core/v3 resources
> - ([Commercial feature][268]) Implemented an active poller for Postgres configuration changes.
> - Print the correct round robin scheduler source (etcd or postgres).

Reviewer: ```suggestion
- ([Commercial feature][268]) Fixed a backend crash when pruning core/v3 resources
- ([Commercial feature][268]) Implemented an active poller for PostgreSQL configuration changes.
- The correct round robin scheduler source ("etcd" or "postgres") is now printed in events and logs.
```

---

## 6.7.2 release notes

**May 11, 2022** &mdash; The latest release of Sensu Go, version 6.7.2, is now available for download.

Sensu Go 6.7.2 delivers several improvements and fixes to the sensu-backend stability and the postgres datastore.

Read the [upgrade guide][1] to upgrade Sensu to version 6.7.2.

**IMPROVEMENTS**

> - ([Commercial feature][268]) In the web UI, we updated the Sensu Plus dialog copy to warn users not to use the form when they have an existing Sumo Logic account.
> - ([Commercial feature][268]) In the web UI, Sensu Plus post setup dialog Source URL copy button now uses a toast notification.

Reviewer: ```suggestion
- ([Commercial feature][268]) In the web UI, the Sensu Plus modal dialog now directs users who already have a Sumo Logic account to follow the instructions to manually set up Sensu Plus.
- ([Commercial feature][268]) In the web UI, the Sensu Plus post-setup modal dialog now indicates success when you use the Copy button to copy the Source URL.
```

- Added the etcd-unsafe-no-fsync backend configuration option, which makes it possible to run sensu-backend with an embedded etcd node for testing and development without placing too much load on the file system.
- Upgraded etcd version from 3.5.2 to 3.5.4.

> 
> **FIXES**
> - ([Commercial feature][268]) Fixed a backend crash when pruning core/v3 resources

Reviewer: ```suggestion

**FIXES**
- ([Commercial feature][268]) Fixed a bug that could cause a backend crash when pruning SumoLogicMetricsHandler and TCPStreamHandler resource types.
```

---

## 6.7.2 release notes

> **May 11, 2022** &mdash; The latest release of Sensu Go, version 6.7.2, is now available for download.

Reviewer: ```suggestion
**May 12, 2022** &mdash; The latest release of Sensu Go, version 6.7.2, is now available for download.
```

Sensu Go 6.7.2 includes a fix for sensu-backend stability and adds an active poller for PostgreSQL config changes.
We've also improved the Sensu Plus modal in the web UI.

Read the [upgrade guide][1] to upgrade Sensu to version 6.7.2.

**IMPROVEMENTS**
- ([Commercial feature][268]) In the web UI, the Sensu Plus modal dialog now directs users who already have a Sumo Logic account to follow the instructions to manually set up Sensu Plus.

> - ([Commercial feature][268]) In the web UI, the Sensu Plus post-setup modal dialog now indicates success when you use the Copy button to copy the Source URL.

Reviewer: ```suggestion
- ([Commercial feature][268]) In the web UI, the Sensu Plus post-setup modal dialog now indicates success when you use the Copy button to copy the Source URL.
- ([Commercial feature][268]) Added supported packages for the Sensu backend, Sensu agent, and sensuctl for Ubuntu 22.04.
```
